RELEASE/scripts/autoscend/quests/level_01.ash

```diff
@@ -0,0 +1,27 @@
+void tootOriole()
+{
+	//Toot Oriole must be visited each ascension to unlock other quests from the council
+	if(get_property("questM05Toot") == "finished")
+	{
+		return;
+	}
+	
+	visit_url("tutorial.php?action=toot");		//do quest
+	//finishing toot quest is not correctly noticed by mafia. r20655 has workaround of correcting this by refreshing quests
+	cli_execute("refresh quests");
+	
+	if(get_property("questM05Toot") == "finished")
+	{
+		use(item_amount($item[Letter From King Ralph XI]), $item[Letter From King Ralph XI]);
```

Comment:
Ed gets Letter to Ed the Undying not Letter From King Ralph XI so this needs tested and verified in Ed before it should be merged otherwise this will completely break HC Ed.